mlc_llm/core.py

```diff
@@ -146,6 +147,14 @@ class BuildArgs:
             "action": "store_true",
         },
     )
+    use_cublas: bool = field(
+        default=False,
+        metadata={
+            "help": ("Offload matmul to cuBLAS when target is CUDA and TVM has been built"
+                     "with cuBLAS enbaled."),
```

Comment:
We should document that cublas offload is meant only for q0f16.

Reply:
Updated